src/ui/actions/timeline.ts

```diff
@@ -99,7 +149,7 @@ function setRecordingDescription({ duration, lastScreen }: RecordingDescription)
 
     // Paint the last screen to get it up quickly, even though we don't know yet
     // which execution point this is and have warped here.
-    paintGraphics(lastScreen);
+    // paintGraphics(lastScreen);
```

Comment:
Well, I thought jumping to the end of the recording and then jumping back to the first comment or meaningful paint could be confusing. But this `paintGraphics()` call doesn't do anything anyway because the description currently doesn't contain a `lastScreen` property

Reply:
cool. then perhaps we should delete the code